src/event-bus/__tests__/EventBus.test.ts

```diff
@@ -0,0 +1,44 @@
+import EventBus from '../EventBus';
+
+describe('EventBus tests', () => {
+    let eventBus: EventBus;
+    const l1 = jest.fn();
+    const l2 = jest.fn();
+    beforeEach(() => {
+        eventBus = new EventBus();
+        jest.clearAllMocks();
+    });
+    test('when dispatch is invoked then all listeners are called', async () => {
```

Comment:
nit: consider adding test to subscribe to different topics

Reply:
Added an extra test